.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+# See https://pre-commit.com for more information
```

Comment:
@scealiontach : did you mean to check in all the pre-commit hooks?

Reply:
yeah. If you don't have pre-commit installed in your local repo. it doesn't do anything.  It's handy though so we recommend it.